src/components/Header.jsx

```diff
@@ -6,8 +6,7 @@ import {
   MdMenu,
 } from "react-icons/md";
 import { Link } from "react-router-dom";
-import { useAuth } from "../contexts/AuthContext";
-import { useTheme } from "../contexts/ThemeContext";
+import { useAuth, useTheme } from "../contexts/contexts";
 import "../styles/header.css";
```

Comment:
Please add a jsconfig.json file and use absolute imports

Reply:
Done